agent/provider/tests/manifest_negotiator.rs

```diff
@@ -302,35 +302,29 @@ fn manifest_negotiator_test_with_node_identity(
 }
 
 #[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "all", "audited-payload": {"default": {"mode": "all", "description": ""}}}}"#, // rulestore config
+    r#"{"outbound": {"enabled": true, "everyone": "all"}}"#, // rulestore config
     r#"["https://domain.com"]"#, // compManifest.net.inet.out.urls
     None; // error msg
     "Accepted because everyone is set to all"
 )]
 #[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "whitelist", "audited-payload": {"default": {"mode": "all", "description": ""}}}}"#, // rulestore config
+    r#"{"outbound": {"enabled": true, "everyone": "whitelist"}}"#, // rulestore config
     r#"["https://domain.com"]"#, // compManifest.net.inet.out.urls
     None; // error msg
     "Accepted because everyone whitelist is matching"
 )]
 #[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "whitelist", "audited-payload": {"default": {"mode": "all", "description": ""}}}}"#, // rulestore config
+    r#"{"outbound": {"enabled": true, "everyone": "whitelist"}}"#, // rulestore config
     r#"["https://non-whitelisted.com"]"#, // compManifest.net.inet.out.urls
-    Some("Audited-Payload rule: Invalid signature"); // error msg
+    Some("Outbound rejected because: Everyone rule didn't match whitelist ; Audited-Payload rule: Invalid signature. ; Partner verification of node descriptor failed: Cannot verify node descriptor structure, schema is not defined ;"); // error msg
```

Comment:
We don't care about partner error message here

Reply:
Done.